root/view/src/main/java/at/fhv/itb5c/view/user/UserModel.java

```diff
@@ -0,0 +1,145 @@
+package at.fhv.itb5c.view.user;
+
+import java.rmi.RemoteException;
+import java.time.LocalDate;
+import java.util.HashSet;
+
+import at.fhv.itb5c.commons.dto.rmi.IUserRMI;
+import at.fhv.itb5c.commons.enums.Gender;
+import at.fhv.itb5c.commons.enums.TypeOfSport;
+import at.fhv.itb5c.commons.enums.UserRole;
+import at.fhv.itb5c.view.util.AlertUtil;
+import javafx.beans.property.DoubleProperty;
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleDoubleProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableSet;
+
+public class UserModel {
+	private StringProperty _firstName;
+	private StringProperty _lastName;
+	private StringProperty _address;
+	private StringProperty _eMail;
+	private StringProperty _telephonenumber;
+	private Gender _gender;
+	private ObjectProperty<LocalDate> _birthDate;
+	private ObservableSet<TypeOfSport> _typeOfSports;
+	private DoubleProperty _membershipFee;
+	private ObservableSet<UserRole> _userRoles;
+
+	private IUserRMI _rmiUser;
+	
+	private UserModel(IUserRMI user) throws RemoteException {
+		_firstName = new SimpleStringProperty();
+		_lastName = new SimpleStringProperty();
+		_address = new SimpleStringProperty();
+		_telephonenumber = new SimpleStringProperty();
+		_eMail = new SimpleStringProperty();
+		_birthDate = new SimpleObjectProperty<>();
+		_membershipFee = new SimpleDoubleProperty();
+		
+		_firstName.setValue(user.getFirstName());
```

Comment:
user may be null. if(user==null) return null; or something like that would be good.